repos/system_upgrade/common/actors/checkconsumedassets/libraries/check_consumed_assets.py

```diff
@@ -69,13 +86,16 @@ def report_incompatible_assets(assets):
         summary_lines += compose_summary_for_incompatible_assets(incompatible_assets, reason)
 
         for asset in incompatible_assets:
-            doc_url_to_title[asset.docs_url].append(asset.docs_title)
+            if asset.docs_url:
+                # Add URLs only when they are specified. docs_url could be empty string
+                doc_url_to_title[asset.docs_url].append(asset.docs_title)
 
     report_parts = [
         reporting.Title(title),
         reporting.Summary('\n'.join(summary_lines)),
         reporting.Severity(reporting.Severity.HIGH),
-        reporting.Groups([reporting.Groups.INHIBITOR, reporting.Groups.REPOSITORY]),
+        reporting.Remediation(hint=_get_hint()),
+        reporting.Groups([reporting.Groups.INHIBITOR, reporting.Groups.SANITY]),
     ]
 
     report_parts += make_report_entries_with_unique_urls(docs_url_to_title_map)
```

Comment:
I went too quick and messed that up.  This is what I meant:

```suggestion
        reporting.Groups([reporting.Groups.INHIBITOR, reporting.Groups.SANITY]),
        make_report_entries_with_unique_urls(docs_url_to_title_map),
    ]

```

Reply:
discussed via slack that this would lead to a unexpected nested list inside `report_parts` of having a flat list. Both agreed to keep it rather unchanged.